docs/spec/api.md.tmpl

```diff
@@ -1000,8 +1000,12 @@ to last response or be fully omitted, depending on the server implementation.
 It may be necessary to list all of the tags under a given repository. The tags
```

Comment:
I missed this earlier: make sure to create a change entry at the top of the specification.